teku/src/main/java/tech/pegasys/teku/cli/BeaconNodeCommand.java

```diff
@@ -149,6 +152,9 @@
   @Mixin(name = "Data")
   private DataOptions dataOptions;
 
+  @Mixin(name = "DataStorage")
```

Comment:
This is a human friendly name so:
```suggestion
  @Mixin(name = "Data Storage")
```